prow/crier/reporters/gerrit/reporter_test.go

```diff
@@ -66,6 +69,7 @@ func (f *fgc) SetReview(instance, id, revision, message string, labels map[strin
 	if len(labels) > 0 {
 		f.reportLabel = labels
 	}
+	f.count++
```

Comment:
Nit: This will be problematic in the future if we add any concurrency tests that operate on multiple PRs. This if fine for now though.